scippy/CMakeLists.txt

```diff
@@ -9,3 +9,10 @@ set(PY_FILES __init__.py table.py plot.py)
 install(TARGETS scippy DESTINATION "scippy")
 install(FILES ${PY_FILES} DESTINATION "scippy")
 install(FILES "compat/mantid.py" DESTINATION "scippy/compat")
+file(GLOB test_files
+    "test/*.py"
+)
+foreach(test_file ${test_files})
+    get_filename_component(test_name ${test_file} NAME_WE)
+    add_test("scippy/${test_name}" "python3" ${test_file})
+endforeach()
```

Comment:
👍 Good idea to add this.

What happens if a developer has a build that is not being installed? For example, I often have debug builds or experimental builds that are not installed.

Reply:
Yes. That would be a big problem, and one of the main reasons I'm on the fence with this PR.